frontend/src/app/components/filters/query-filter/query-filter.component.ts

```diff
@@ -34,6 +34,7 @@ import {AngularTrackingHelpers} from 'core-components/angular/tracking-functions
 import {QueryFilterInstanceResource} from "core-app/modules/hal/resources/query-filter-instance-resource";
 import {BannersService} from "core-app/modules/common/enterprise/banners.service";
 import {WorkPackageViewFiltersService} from "core-app/modules/work_packages/routing/wp-view-base/view-services/wp-view-filters.service";
+import {SchemaCacheService} from "core-components/schemas/schema-cache.service";
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/opf/openproject/blob/3f0519806a21698d886f0aae816d7a75ed1548e8/frontend/src/app/components/filters/query-filter/query-filter.component.ts#L38"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</del></td>
    </tr>
  </tbody>
</table>